### ISSUE TITLE
Ignore multiple lines

### ISSUE THREAD

**Issue by RMeli** [MarcoGorelli/cython-lint#17]
It would be useful to have, in addition to `# no-cython-lint` to ignore a single line, to have something like

```python
# cython-lint off
pass
# cython-lint on
```

to ignore multiple lines of code.

**Comment by MarcoGorelli** [comment]
Hi @RMeli - what's the use-case for this?

**Comment by MarcoGorelli** [comment]
gonna say no for now, not really keen on adding complexity, but thanks anyway for your request

**Comment by RMeli** [comment]
Hi @MarcoGorelli, the use case I had in mind is to turn off linting for multiple lines of code, where specific formatting is required/preferred (long mathematical expressions, arrays, ...). Some tools allow this (especially formatting tools, such as `black`), but I totally understand keeping the complexity low.
